partner_credit_limit/model/account_invoice.py

```diff
@@ -10,10 +10,16 @@ class AccontInvoice(models.Model):
     _inherit = 'account.invoice'
 
     @api.multi
-    def check_limit_credit(self):
+    def check_payment_type(self):
         for invoice in self:
             if invoice.payment_term_id.payment_type != 'credit':
                 return True
+
+    @api.multi
+    def check_limit_credit(self):
+        for invoice in self:
+            if invoice.check_payment_type():
```

Comment:
Could you adapt with filtered?

Reply:
@JulioSerna the change it was made in [5b2026d](https://github.com/Vauxoo/addons-vauxoo/pull/1260/commits/5b2026df870cf2c7a70fd4e0ab3316451f194aff)